### ISSUE TITLE
Fix variants regexes for AA translation

### ISSUE THREAD

**Issue by Colossus** [ajratner/dd-genomics#226]
Currently, Asn is sometimes translated as A (Alanine) because the regex picks up only the first A --- in fact, should be translated as N (Asparagine)

**Comment by amwenger** [comment]
I bet that no one will mix the three-letter and one-letter amino acid codes.  You'll see for example `Asn263Glu` and `N263G` but not `N263Glu` or `Asn263G`.  It sounds like your regex is matching `Ttt###O` which I imagine is a mistake in most cases.

**Comment by Colossus** [comment]
Yeah that's what I thought. That's exactly what's happening. I just have to sort it out.
